app/controllers/concerns/search_autocomplete_ab_testable.rb

```diff
@@ -0,0 +1,31 @@
+module SearchAutocompleteAbTestable
+  ALLOWED_VARIANTS = %w[A B Z].freeze
+
+  def self.included(base)
+    base.helper_method(
+      :search_autocomplete_ab_test_variant,
+      :show_search_autocomplete?,
+    )
+    base.after_action :set_search_autocomplete_ab_test_response_header
+  end
+
+  def search_autocomplete_ab_test
+    @search_autocomplete_ab_test ||= GovukAbTesting::AbTest.new(
+      "SearchAutocomplete",
+      allowed_variants: ALLOWED_VARIANTS,
+      control_variant: "Z",
```

Comment:
Minor: I thought the control variant was A? 
https://github.com/alphagov/govuk-fastly/blob/main/dictionaries.yaml#L17

Reply:
Ah. This is the old debate from the other week around my usage of the word "control" 🥲 

_Technically_, A is the control and Z is the baseline, however, the terminology in the GOV.UK AB testing library is different. In Finder Frontend, we also [declare Z as the control](https://github.com/alphagov/finder-frontend/blob/main/app/controllers/ab_test/search_autocomplete_ab_testable.rb#L10) for this test.